api/preprints/urls.py

```diff
@@ -12,4 +12,5 @@
     url(r'^(?P<preprint_id>\w+)/identifiers/$', views.PreprintIdentifierList.as_view(), name=views.PreprintIdentifierList.view_name),
     url(r'^(?P<preprint_id>\w+)/contributors/$', views.PreprintContributorsList.as_view(), name=views.PreprintContributorsList.view_name),
     url(r'^(?P<preprint_id>\w+)/review_actions/$', views.PreprintActionList.as_view(), name=views.PreprintActionList.view_name),
+    url(r'^(?P<preprint_id>\w+)/requests/$', views.PreprintRequestListCreate.as_view(), name=views.PreprintActionList.view_name),
```

Comment:
There might be some copypasta here. Should be `name=views.PreprintRequestListCreate.view_name`. Discovered this while working on the reviews detail page, which would make a request for a preprint detail with embedded `review_actions`.

Reply:
Also, should `PreprintSerializer` maintain a relationship field to `PreprintRequestListCreate` view? Basically, there should be an easier way for the front end to query the related requests of a preprint, like an embeddable relationship field.

Reply:
A relationship to any existing withdrawal requests is not on the `PreprintSerializer` because the existence of any such requests isn't public information -- only moderators should be able to see them.